cmd/envtool/main.go

```diff
@@ -277,18 +277,17 @@ func run(ctx context.Context, logger *zap.SugaredLogger) error {
 	return nil
 }
 
-func main() {
-	debugF := flag.Bool("debug", false, "enable debug mode")
-	flag.Parse()
+// The cli struct represents all command-line commands, fields and flags.
```

Comment:
```suggestion
// cli struct represents all command-line commands, fields and flags.
```

to make it like everywhere (not super critical I guess).